docs/instruction_for_testing.md

```diff
@@ -0,0 +1,135 @@
+# How to run full integration test suite
+
+The purpose of this document is to detail the steps to run the project's integration tests on each supported platform. Not all test flows have been integrated in our automated pipeline so it is important to follow this guide and understand how to run tests in a semi-automatically fashion.
+
+It is **important** to note that this guide is not strictly for running tests while prepping for a release. This guide should be reviewed and the testing process defined here should be run before a release, as you incrementally add logic. That way, if bugs are uncovered, you have enough time to address them!
+
+## Deployment and Setup
+
+Before we can run our integration tests, we need to deploy the required environments and create the test artifact.
+
+At a high level, you will need to:
+
+1. Deploy and configure machines
+1. Install dependencies
+1. Build the `integrations_tests_runner`
+
+#### Deploy and configure machines
+
+You will need a machine to host your Conjur server (should be Appliance) and 4 different EC2 machines (Windows, RHEL 7, RHEL 8, macOS - if available) for packing the executable. If macOS is not available, your local macOS machine should suffice.
+
+#### Install dependencies
+
+On those 4 machines, clone the project's repostiory and install dependencies. To install dependencies,  follow the instructions [here](https://github.com/cyberark/conjur-api-python3/blob/master/CONTRIBUTING.md#development).
+
+#### Build the `integrations_tests_runner`
+
+The test runner executable wraps the CLI and the integration tests into a single executable, allowing us to run our integration tests without installing additional dependencies except for what the CLI requires. For more information see [here](https://github.com/cyberark/conjur-api-python3/blob/master/CONTRIBUTING.md#running-tests-outside-of-a-containerized). 
```

Comment:
```suggestion
The test runner executable wraps the CLI and the integration tests into a single executable, allowing us to run our integration tests without installing additional dependencies (**including python**) except for what the CLI requires. For more information see [here](https://github.com/cyberark/conjur-api-python3/blob/master/CONTRIBUTING.md#running-tests-outside-of-a-containerized). 
```

Reply:
Remove the link since it exist in this doc as well.

Reply:
Done